### PR TITLE
web3.sha3 test

### PR DESCRIPTION
**Pull request by debris**
web3.sha3 test (not passing)

### CODE REVIEW

File: test/webthreestubclient.h [hunk 1]
```diff
@@ -12,6 +12,16 @@ class WebThreeStubClient : public jsonrpc::Client
     public:
         WebThreeStubClient(jsonrpc::IClientConnector &conn) : jsonrpc::Client(conn) {}
 
+        std::string web3_sha3(const std::string& param1) throw (jsonrpc::JsonRpcException)
```
Comment (CJentzsch) [inline test/webthreestubclient.h]: throw as exception specification is deprecated. See here for more infos: http://stackoverflow.com/questions/13841559/deprecated-throw-list-in-c11
I would just add a comment.

File: test/webthreestubclient.h [hunk 2]
```diff
@@ -12,6 +12,16 @@ class WebThreeStubClient : public jsonrpc::Client
...
+        {
+            Json::Value p;
+            p.append(param1);
+            Json::Value result = this->CallMethod("web3_sha3",p);
+            if (result.isString())
+                return result.asString();
+            else
+                throw jsonrpc::JsonRpcException(jsonrpc::Errors::ERROR_CLIENT_INVALID_RESPONSE, result.toStyledString());
```
Comment (CJentzsch) [inline test/webthreestubclient.h]: Usually we use boost exceptions throughout the codebase. Make JsonRpcException derive from boost::exception and throw like that: BOOST_THROW_EXCEPTION(jsonrpc::JsonRpcException)

Comment (debris) [inline test/webthreestubclient.h]: that's autogenerated code :)

File: test/webthreestubclient.h (hunk 1 above)
Comment (debris) [inline test/webthreestubclient.h]: that's autogenerated code :)

**Comment by debris** [comment]
test is passing now, the problem was with the wrong padding in fromAscii i fixed that also in [47687cf085196e2e4fa7ec42ff426f241dae10c5](https://github.com/ethereum/ethereum.js/commit/47687cf085196e2e4fa7ec42ff426f241dae10c5)

**Pull request merged**